src/index.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+const state = {
+  tempCount: 49,
+  city: '✨✨✨',
+  sky: 'Sunny',
+};
+
+const convertKelvinToFarenheit = (kelvinTemp) => {
+  return Math.floor((kelvinTemp - 273.15) * (9 / 5) + 32);
+};
+
+const changeTempColor = () => {
+  let temp = state.tempCount;
+  if (temp <= 49) {
+    document.getElementById('tempCounter').style.color = 'teal';
+  } else if (temp <= 59) {
+    document.getElementById('tempCounter').style.color = 'green';
+  } else if (temp <= 69) {
+    document.getElementById('tempCounter').style.color = 'yellow';
+  } else if (temp <= 79) {
+    document.getElementById('tempCounter').style.color = 'orange';
+  } else {
+    document.getElementById('tempCounter').style.color = 'red';
+  }
+};
+
+const changeBackground = () => {
+  let temp = state.tempCount;
+  document.body.style.backgroundSize = '100% 900px';
+  if (temp <= 59) {
+    document.body.style.backgroundImage = "url('./assets/59_or_below.jpeg')";
+  } else if (temp <= 69) {
+    document.body.style.backgroundImage = "url('./assets/69_or_below.jpeg')";
+  } else if (temp <= 79) {
+    document.body.style.backgroundImage = "url('./assets/79_or_below.jpeg')";
+  } else {
+    document.body.style.backgroundImage = "url('./assets/80_and_above.png')";
+  }
+};
+
+const addDegree = () => {
+  state.tempCount += 1;
+  const tempCounter = document.querySelector('#tempCounter');
+  tempCounter.textContent = `${state.tempCount} \u00B0`;
+  changeTempColor();
+  changeBackground();
+  console.log(state.tempCount);
+};
+const removeDegree = () => {
+  state.tempCount -= 1;
+  const tempCounter = document.querySelector('#tempCounter');
+  tempCounter.textContent = `${state.tempCount} \u00B0`;
+  changeTempColor();
+  changeBackground();
+};
+
+const updateCity = () => {
+  if (document.querySelector('#city').value) {
+    state.city = document.querySelector('#city').value;
+  }
+  const cityDisplay = document.querySelector('#cityDisplay');
+  cityDisplay.textContent = `for the lovely city of ${state.city}`;
+};
+
+const resetCity = () => {
+  state.city = '✨✨✨';
+  document.querySelector('#city').value = '';
+  const cityDisplay = document.querySelector('#cityDisplay');
+  cityDisplay.textContent = `for the lovely city of ${state.city}`;
+};
```

Comment:
Can you think of how you could combine these functions to make your code more DRY?